crates/daktilo_lib/Cargo.toml

```diff
@@ -16,7 +16,7 @@ rust-version = "1.74.1"
 audio-tests = []
 
 [dependencies]
-rdev = { version = "0.5.3", features = ["serialize"] }
+rdev = { git = "https://github.com/Narsil/rdev.git", rev = "c77b4e5", features = ["serialize", "wayland"] }
```

Comment:
Unfortunately we can't release crates to crates.io with git dependencies... do you know if the latest version of the crate is released or when is it going to be released?

Reply:
Not yet, I'm gonna ask.

Reply:
Thanks!